### PR TITLE
Add data manager to download NCBI taxonomy

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by davebx** [comment]
XREF: https://github.com/nekrut/tools-devteam/pull/3

**Comment by bgruening** [comment]
General question. Do we need to have the name field here, or can we simply default to a readable date format?
From a usability point of view the only thing I care is the fetch-date? Right?

@blankenberg @davebx what do you think?
I would always enter the date into the name field :)

**Comment by davebx** [comment]
@bgruening That's essentially how I was picturing the name field being used, as a place for a galaxy admin to enter the download date.

File: data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<tool id="ncbi_taxonomy_fetcher" name="NCBI" tool_type="manage_data" version="1.0.0">
+    <description>taxonomy downloader</description>
+    <stdio>
+        <exit_code description="Error" level="fatal" range="1:" />
+    </stdio>
+    <command>
+    <![CDATA[
+        curl -o taxdump.tar.gz ${taxonomy_url} && tar xzf taxdump.tar.gz && python $__tool_directory__/data_manager.py --name "${database_name}" --out "${out_file}"
+    ]]>
```
Comment (bgruening) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: `curl` is not installed on every system, so I think we need to depend on https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_libcurl_7_35/tool_dependencies.xml

Comment (davebx) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: Does that install the curl executable as well?

**Comment by bgruening** [comment]
@davebx can we simply default to str(datetime.date.today()) if the user has not entered anything, with a nice help text under this parameter?

File: data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml (hunk 1 above)
Comment (bgruening) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: Very good question, let see if I have it installed on Galaxy-FR.

Comment (bgruening) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: Ha, have it and yes in /bin there is a curl binary.

Comment (bgruening) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: We should remove the ssl dependency on this package.

Comment (davebx) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml]: Awesome, I'll update the dependencies right away.

**Comment by davebx** [comment]
@bgruening Good idea.

**Comment by bgruening** [comment]
:+1: Thanks @davebx and sorry for the delay.

**Pull request merged**